### ISSUE TITLE
Enable deeplinking to transfers

### ISSUE THREAD

**Issue by jmrossy** [hyperlane-xyz/hyperlane-warp-ui-template#35]
#30 and #31 are prerequisites 

**Current behavior:**

No way to access transfer details directly via a link

**Desired behavior:**

Transfer data is encoded into a URL and deep-links auto-display the transfer details modal. This must work for both the user that initiated the transfer and users that did not.

**Comment by avious00** [comment]
@jmrossy does this make sense to pull into this sprint?

**Comment by jmrossy** [comment]
@avious00 I don't think so. Personally I don't think this feature is worthwhile but Nam was in favor
